app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,10 @@
+class OrdersController < ApplicationController
+  def create
+    @client = current_user
+    @line_items = @client.order.last.line_items
```

Comment:
instance variables can be pluralized, that just means you see them in the view. Like for the reddit thing we made I assume you had something like @messages = Message.all to make an index. I did mess up the implementation of getting the total and stuff though. The method is supposed to be on the order and pass the @line_items array in as an argument so I will fix that tomorrow. Kinda embarrassing